### PR TITLE
Breadcrumbs: switching from > to glyhpicon

### PR DESCRIPTION
**Pull request by Garneauma**
Changes related to WET-325

### CODE REVIEW

**Comment by Garneauma** [comment]
Demo link: https://servicecanada.github.io/wet-boew-demos/gcweb-pr2122/sites/breadcrumbs/breadcrumbs-en.html

**Review by GormFrank — CHANGES_REQUESTED** [review]
Suggestion to add a working example section with no link, but mentioning to look at the top of the current page.

And can you update the JSON-LD index page to reflect the name change?

**Comment by duboisp** [comment]
I will perform a full review to have the PR ready to merge next week

File: sites/breadcrumbs/breadcrumbs-en.html [hunk 1]
```diff
@@ -5,46 +5,142 @@
...
+<table class="table table-bordered">
+	<tr>
+		<th style="width:0;">Breaking</th>
+		<td>n/a</td>
+	</tr>
+	<tr>
+		<th>Additions</th>
+		<td>
+			<ul>
+				<li>Styles: changing levels separator from a greater-than sign (&gt;) to a chevron right icon (<span class="glyphicon glyphicon-chevron-right"></span>)</li>
```
Comment (duboisp) [inline sites/breadcrumbs/breadcrumbs-en.html:57]: It is an addition or more that change is a fix from a perspective of the web author/web implementer.

And from a user perspective, this change is low impact compared to the preceding pattern when evaluated from the cognitive challenges (visually and functional) of using this component

File: sites/breadcrumbs/breadcrumbs-en.html [hunk 2]
```diff
@@ -5,46 +5,142 @@
...
+<p>Each page or section name on the Canada.ca site should be associated with a shortened version of the name, for the purposes of breadcrumb and other navigation systems.</p>
+<p>For example, when on the “Planning a business” page in the Business and industry theme, the breadcrumb trail will be:</p>
+<p class="mrgn-tp-md">Canada.ca&nbsp;&nbsp; &gt;&nbsp;&nbsp; Business&nbsp;&nbsp; &gt;&nbsp;&nbsp; Starting a business</p>
+<p><strong>Note</strong>: GCWeb vX.0 are no longuer compatible with the pre 2019 megamenu pattern (GCWEB v4.x) that was taking the full page width.</p>
+<p><strong>Recommendation</strong>: The page title should not be included within the breadcrumbs list items.</p>
+
+<h4>Code sample</h4>
+	<p>Here is the html output with structured data in RDFa format for the above breadcrumb example:</p>
+	{%- highlight html -%}
+	<nav id="wb-bc" property="breadcrumb">
```
Comment (duboisp) [inline sites/breadcrumbs/breadcrumbs-en.html:86]: Is this example are the same as the iteration 1 bellow? I looked and I don't see the difference.

What about "Code sample (Iteration 1)"

File: sites/breadcrumbs/breadcrumbs-fr.html [hunk 3]
```diff
@@ -6,46 +6,142 @@
...
+<table class="table table-bordered">
+	<tr>
+		<th style="width:0;">Majeurs</th>
+		<td>n/a</td>
+	</tr>
+	<tr>
+		<th>Additions</th>
+		<td>
+			<ul>
+				<li>Styles: Modification du séparateur de niveaux d'un signe «&nbsp;plus grand que&nbsp;» (&gt;) à une icône de chevron à droite (<span class="glyphicon glyphicon-chevron-right"></span>)</li>
```
Comment (duboisp) [inline sites/breadcrumbs/breadcrumbs-fr.html:58]: Is this change are an addition or a fix?

File: sites/breadcrumbs/breadcrumbs-fr.html [hunk 4]
```diff
@@ -6,46 +6,142 @@
...
+	<li>Utilisez une icône de chevron à droite (<span class="glyphicon glyphicon-chevron-right"></span>) pour séparer les niveaux</li>
+	<li>Les longs titres de page doivent être raccourcis dans la mesure du possible, pour permettre de réduire l’espace horizontal nécessaire pour afficher la piste.</li>
+</ul>
+<p>Le nom de chaque page ou section dans le site Canada.ca doit être associé à une version raccourcie du nom, aux fins du fil d'Ariane et d’autres systèmes de navigation. </p>
+<p>Par exemple, lorsqu'une personne est sur la page «&nbsp;Planification d'entreprise&nbsp;» dans le thème «&nbsp;Entreprises et industrie&nbsp;» le fil d'Ariane sera&nbsp;:</p>
+<p>Canada.ca &gt; Entreprises &gt; Lancer une entreprise</p>
+<p><strong>Note</strong>: GCWeb vX.0 n'est plus compatible avec le modèle de mégamenu antérieur à 2019 (GCWEB v4.x) qui prenait toute la largeur de la page.</p>
+<p><strong>Recommendation</strong>: Le titre de la page ne dois pas être inclus parmis la liste de fil d'arianne.</p>
+
+<h4>Exemple de code</h4>
```
Comment (duboisp) [inline sites/breadcrumbs/breadcrumbs-fr.html:84]:
```suggestion
<h4>Exemple de code (iteration 1)</h4>
```

**Review by duboisp — CHANGES_REQUESTED** [review]
Content change required which describe the type of change applied.

* [ ] Review the type of change to ensure it is accurate from our public versioning API perspective

Optional (can be done later)
* [ ] Remove the override of the CSS attribute "content" by wet-boew if possible

File: sites/breadcrumbs/breadcrumbs-en.html (hunk 2 above)
Comment (Garneauma) [reply sites/breadcrumbs/breadcrumbs-en.html:86]: The code sample is the same as iteration 1 since there were no HTML changes in iteration 2.

**Pull request merged**